conf/test_hybrid_host_rm.config

```diff
@@ -10,7 +10,7 @@
 params {
   config_profile_name = 'Test profile'
   config_profile_description = 'Minimal test dataset to check pipeline function'
-  // Limit resources so that this can run on GitHub Actions
+  // Limit resources so that this can run on GitHub Actionss
```

Comment:
It is "Actions" and not "Actionss", is it?

Reply:
Thanks! :)